librosa/display.py

```diff
@@ -451,6 +455,10 @@ def waveplot(y, sr=22050, max_points=5e4, x_axis='time', offset=0.0,
     target_sr = sr
     hop_length = 1
 
+    # Pad an extra channel dimension, if necessary
+    if y.ndim == 1:
+        y = util.stack([y], axis=0)
```

Comment:
Neat! So this turns `(n_samples,)` into `(1, n_samples)` without breaking F-contiguity? Is there a reason why this couldn't be achieved with `.reshape(1, -1)`?

Reply:
ah good catch; reshape would be more efficient here